www/app/views/partials/md-table.tmpl.html

```diff
@@ -16,7 +16,7 @@
       <td class="md-table-content" ng-click="go(c)" ng-repeat="h in headers" ng-class="customClass[h.field]" ng-if="h.field != thumbs">
         {{c[h.field]}}
       </td>
-      <td class="md-table-td-more">
+      <td class="md-table-td-more" ng-if="doMore()">
```

Comment:
Yes, that was intentional to disable the more link functionality until explicitly enabled. I've since updated the code to check for the moreClick handler instead.